index.html

```diff
@@ -6,45 +6,75 @@
     <link rel="stylesheet" type="text/css" href="styles.css">
     <div class="t_b">
         <h2 id="Title_Heading">NEWSFEED</h2>
-        <h6 id="Title_Desc" >Yet another newsfeed</h6>
+        <h6 id="Title_Desc">Yet another newsfeed</h6>
     </div>
 </head>
 
 <body>
-<div class = post>
```

Comment:
Always maintain a wrapper, for example
<div class="wrapper">
<div class="post"></div>
</div>